js/packages/web/src/actions/getStorefront.ts

```diff
@@ -19,7 +19,7 @@ export const getStorefront = async (
       body: JSON.stringify({
         query: `
             query GetStorefrontTheme($subdomain: String!) {
-              transactions(tags:[{ name: "holaplex:metadata:subdomain", values: [$subdomain]}], first: 1) {
+              transactions(tags:[{ name: "holaplex:metadata:subdomain", values: [$subdomain]}], sort: HEIGHT_DESC, first: 1) {
```

Comment:
This is the default.

Reply:
Okay, that's what I thought.  Arweave docs were a little coy on that.